x-pack/plugins/search_playground/server/routes.ts

```diff
@@ -240,4 +240,62 @@ export function defineRoutes({
       });
     })
   );
+
+  router.post(
+    {
+      path: APIRoutes.POST_SEARCH_QUERY,
+      validate: {
+        body: schema.object({
+          search_query: schema.string(),
+          elasticsearch_query: schema.string(),
+          indices: schema.arrayOf(schema.string()),
+          size: schema.maybe(schema.number({ defaultValue: 10, min: 0 })),
+          from: schema.maybe(schema.number({ defaultValue: 0, min: 0 })),
+        }),
+      },
+    },
+    errorHandler(logger)(async (context, request, response) => {
+      // TODO validate indices not empty, figure out if it is ok
+      const { client } = (await context.core).elasticsearch;
```

Comment:
we should validate that indices are not empty. There is a bug in playground where the session may refer to a non existant index. We have a ticket to fix this.